src/components/Dropzone/index.vue

```diff
@@ -0,0 +1,277 @@
+<template>
+  <div
+    :id="id"
+    :ref="id"
+    :action="url"
+    class="dropzone"
+  >
+    <input
+      type="file"
+      name="file"
+    >
+  </div>
+</template>
+
+<script lang="ts">
+import Dropzone from 'dropzone'
+import 'dropzone/dist/dropzone.css'
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator'
+
+export interface DropzoneFile extends File {
```

Comment:
Is there a similar interface (like DropzoneFile) in Dropzone type definition file?

Reply:
I use  Dropzone.DropzoneFile.